packages/firebase_ml_vision/ios/Classes/TextRecognizer.m

```diff
@@ -85,7 +85,9 @@ + (void)addData:(NSMutableDictionary *)addTo
 
   __block NSMutableArray<NSDictionary *> *allLanguageData = [NSMutableArray array];
   for (FIRVisionTextRecognizedLanguage *language in languages) {
-    [allLanguageData addObject:@{@"languageCode" : language.languageCode}];
+    [allLanguageData addObject:@{
+      @"languageCode" : language.languageCode ? language.languageCode : [NSNull null]
```

Comment:
A crash would occur if `language.languageCode` was `nil` or `NULL` was add to the `NSDictionary`.